terraform-google-{{cookiecutter.module_name}}/test/integration/simple_example/controls/gsutil.rb

```diff
@@ -12,32 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-project_id = attribute('project_id')
+control "gsutil" do
```

Comment:
Non-blocking: I'd suggest we also retain a gcloud example since gcloud tests are far more common than gsutil ones.